descriptors/ModuleDescriptor-template.json

```diff
@@ -116,6 +116,38 @@
         }
       ]
     },
+    {
+      "id": "custom-fields",
```

Comment:
Can you rename `custom-fields` to `users-custom-fields`?
Each module may have custom fields. The id of mod-inventory-storage will probably be `inventory-custom-fields`.

Reply:
Because of `"interfaceType" : "multiple"` the rename is not needed: https://github.com/folio-org/okapi/blob/master/doc/guide.md#multiple-interfaces